aiocouch/remote.py

```diff
@@ -179,6 +186,12 @@ async def _streamed_request(
         async with self._http_session.request(
             method, url=f"{self._server}{path}", **kwargs
         ) as resp:
+            if not resp.ok:
+                # Save the reason for later if any
+                try:
+                    self.failure_reason = (await resp.json())["reason"]
```

Comment:
I'm unfamiliar enough with aiohttp streamed request to know if I'm doing something wrong here. 

Please give your opinion on it I would be grateful.